.github/workflows/maven.yml

```diff
@@ -0,0 +1,56 @@
+name: Java CI
+
+on:
+  push:
+    branches: # reacting on push action to the following branches
+      - develop
+      - 'feature/**'
+
+jobs:
+  prep: # preparation job, retrieving current version from pom and the shortrev from git
+    runs-on: ubuntu-latest
+    outputs:
+      rev: ${{ steps.shortrev.outputs.rev }}
+      ver: ${{ steps.pversion.outputs.ver }}
+      date: ${{ steps.date.outputs.date }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8 & run tests & generate artifacts
+        uses: actions/setup-java@v1
```

Comment:
can you please use v2 here, for all the functions? As described on confluence (maybe even with 2 files as we do for other adapters - for dev and release builds)